stablehlo/tests/ops_stablehlo.mlir

```diff
@@ -3055,6 +3055,47 @@ func.func @dot_general(%arg0: tensor<1x?x1x?xf32>, %arg1: tensor<?x1x?x1x?xf32>)
 
 // -----
 
+// CHECK-LABEL: func @dot_general_algorithm
+func.func @dot_general_algorithm_tf32(%arg0: tensor<2x2x2xi64>, %arg1: tensor<2x2x2xi64>) -> tensor<2x2x2xi64> {
```

Comment:
Let's move the `_tf32` and `_i#` tests below constraint tests for alphabetical ordering

Reply:
Actually... the `_i#` and `_c#` tests shouldn't have `algorithm` in them at all.. Removed that now the positive algorithm test is alphabetically first.

Should I put the input tests after the constraint tests to be alphabetical? `dot_general_i8` vs `dot_general_c1`?

Reply:
Yeah, I also ordered them alphabetically for other test, though I prefer inputs before constraints.

Reply:
ordered alphabetically as well, thanks!